### ISSUE TITLE
Have the first item in the list be highlighted by default

### ISSUE THREAD

**Issue by ykhan21** [LinArcX/wmenu#10]
Currently in wmenu to select an item from the list, you need to first use the arrow keys to highlight what you need, then press Enter to select that item.
Most of the time (after filtering) the needed item is the first one. If the first item were highlighted by default, the arrow key press would be unnecessary.
That way you just search for an item and hit Enter to select it.
It would be nice if this were the default behavior or an option that could be passed. This is the default behavior in fzf.

**Comment by LinArcX** [comment]
You're right. I will implent this feature. Currently i'm re-writing the application in c.
